common/lib/api-client/models/index.js

```diff
@@ -18,7 +18,6 @@ const gender = require('./gender')
 const image = require('./image')
 const journey = require('./journey')
 const location = require('./location')
-const locationsFreeSpaces = require('./locations-free-spaces')
```

Comment:
The `locations-free-spaces` model is not actually used - as `location-free-spaces` service returns `location` models.